client/src/screens/events/Detail.js

```diff
@@ -302,7 +302,6 @@ class Detail extends Component {
     if (!event && !loading) {
       return (
         <Center>
-          <Progress />
```

Comment:
Approved - but what happens when the query from eventQuery is loading now - you want it to display unable to load event?

Reply:
errr no. first time around (when event is undefined) it will show a progress spinner.

Further loading will just be in the background